src/v/pandaproxy/api/api-doc/schema_registry_header.json

```diff
@@ -9,4 +9,4 @@
   "schemes": [
     "http"
   ],
-  "paths": {
+  "paths": {
```

Comment:
This could be in the previous commit

We also typically put newline characters at the end of files.

Reply:
Yeah...I'm sort of trying to follow the contributors doc, so I would squash these all back in at the end. But this tiny commit is not very helpful I think.

>newline characters

In this case there was none to begin with, so I was restoring that. Something to do with concatenation/swagger, but I don't think it breaks anything either way.